### DISCUSSION TITLE
JSON API based on https://laravel-rest-api.lomkit.com/

### DISCUSSION THREAD

**Discussion by aiiddqd** [lunarphp/lunar#1883] [Feature Requests]
I noticed that you are working on the JSON API.
What do you think about this https://laravel-rest-api.lomkit.com/ 
To me this looks like a simple and elegant solution

**Comment by glennjacobs** [comment]
It's something I reviewed recently and haven't dismissed it. We plan to release the Storefront API for version 1.1, so we won't make any final decisions until v1 is fully out.

**Reply by yasirmturk** [reply]
How can I join the working on Storefront API  and also demo iOS app
